src/i18n/config.ts

```diff
@@ -0,0 +1,24 @@
+import i18next from 'i18next';
+import HttpApi from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+const i18n = i18next
+  .use(HttpApi)
+  .use(LanguageDetector)
+  .init({
+    fallbackLng: 'en',
```

Comment:
I get a warning in my browser that there's no `en-US` found... maybe that should be default?

Reply:
Yes, I've seen that happening. That's because there isn't a file specifically for `en-US` or `en-GB` and so on. I am gonna do some reasearch and try to find a way to resolve all `en-SOME_ENGLISH_SPEAKING_COUNTRY` to just `en`. That's gonna be useful for other languages, such as `pt-BR` or `pt-PT`.

Reply:
nice, works!